drivers/eeprom/eeprom.c

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright (C) 2019 Otto-von-Guericke-Universität Magdeburg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup   drivers_eeprom
+ * @{
+ *
+ * @file
+ * @brief     Implementation for EEPROM driver API
+ *
+ * @author    Marian Buschsieweke <marian.buschsieweke@ovgu.de>
+ * @}
+ */
+
+#include <errno.h>
+#include <string.h>
+
+#include "assert.h"
```

Comment:
This is a system include, so should be `#include <assert.h>` and moved with the other system includes above.

Reply:
I have to contradict that unfortunately. This header is from `core/includes` and not from of standard C libraies. IMHO, it is correct to include it with `"assert.h"`.

Reply:
You are both right. On POSIX systems it is a system header. But the standard requests that a failing assert send the signal SIGABORT (not sure about the spelling), so the standard version is not available on embedded systems. Luckily RIOT provides a compatible implementation (form the user's perspective; there is no SIGABORT used for obvious reasons, though).

With that in mind, it is not that clear if it should be treated as system lib or not. I've seen both versions in RIOT. I'm personally fine with either.

Reply:
So, if no one speaks up I just keep it as it is.